Plugzy.Service/Services/AuthenticationService.cs

```diff
@@ -0,0 +1,48 @@
+﻿using Azure.Core;
+using Plugzy.Domain.Entities;
+using Plugzy.Infrastructure.Interface.Repository;
+using Plugzy.Models.Base;
+using Plugzy.Models.Response;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Plugzy.Service.Services
+{
+    public class AuthenticationService : IAuthenticationService
```

Comment:
ayrıca bir authentication servise gerek yok sanki,  her command kendi içinde zaten tek metodu kullanıyor,  onları ilgili command lerin içlerine  taşıyabilirsin.  eğer tekrar eden ve farklı yerlerde kullanılan ortak metotlar olacaksa bir servis yaparak ilerleyebilirsin

Reply:
CheckOtpCode  hem otpcommand hem de authcommand ikisindede çağrılıyor.CheckPhone tek bir yerde kullanıldığı için kaldırılmıştır.